tensorflow_addons/metrics/f_scores.py

```diff
@@ -175,7 +175,8 @@ def __init__(self,
                 initializer='zeros',
                 dtype=self.dtype)
 
-    def update_state(self, y_true, y_pred):
+    # TO DO SSaishruthi: Add sample weight option
```

Comment:
Change this to: 
```
# TODO: Add sample_weight support, currently it is ignored during calculations.
```

Reply:
Updated but can I add my name so that I can take it up after this update?